modules/_labs/color-picker/react/lib/parts/SwatchBook.tsx

```diff
@@ -55,9 +63,9 @@ const SwatchContainer = styled('div')<SwatchContainerProps>(
 );
 
 const Container = styled('div')({
-  display: 'grid',
-  gridTemplateColumns: 'repeat(8, auto)',
-  gridGap: spacing.xxs,
+  display: ['grid', '-ms-grid'],
```

Comment:
```suggestion
  display: ['-ms-grid', 'grid'],
```

It is better to use prefix values first and specification values afterward